src/app/components/input/input.component.html

```diff
@@ -7,9 +7,9 @@
 </div>
 
 <div #dropNote class="form-container active-form" *ngIf="!showFirst">
-  <form class="drop-note">
-    <input type="text" class="note-title" placeholder="Title"/>
-    <input class="note-text" type="text" placeholder="Take a note..."/>
+  <form class="drop-note" [formGroup]="notes" (ngSubmit)="createNote()">
+    <input type="text" class="note-title" placeholder="Title" formControlName="title" />
```

Comment:
I think you should think from the end user side not as a dev so please change the placeholder